packages/mcp-server-supabase/src/platform/types.ts

```diff
@@ -110,6 +110,7 @@ export const deployEdgeFunctionOptionsSchema = z.object({
 
 export const executeSqlOptionsSchema = z.object({
   query: z.string(),
+  parameters: z.array(z.any()).optional(),
```

Comment:
Safer to use `unknown` here (this is also how we type it in postgres-meta/infra)

```suggestion
  parameters: z.array(z.unknown()).optional(),
```